WindowsServerDocs/storage/storage-replica/Stretch-Cluster-Replication-Using-Shared-Storage.md

```diff
@@ -42,6 +42,9 @@ This walkthrough uses the following environment as an example:
 -   A network between servers with enough bandwidth to contain your IO write workload and an average of =5ms round trip latency, for synchronous replication. Asynchronous replication does not have a latency recommendation.
 -   The replicated storage cannot be located on the drive containing the Windows operating system folder.
 
+> [!IMPORTANT]
+> Whilst it is possible to attach a storage device to a single server and use this for replication, Windows Failover Clustering still relies upon SCSI Persistent Reservations. Therefore, the storage must still be a Shared Storage type such as a SAN technology. Local disks or disks presented by a hypervisor may not be compatible. In Azure, the disks must be a Premium SSD size that supports sharing, even if only one VM is to be attached to it.

```

Comment:
Note that "whilst" may sound historical (maybe even archaic) to an English-speaking American person. "While" is sufficient in American English.
```suggestion
> While it is possible to attach a storage device to a single server and use this for replication, Windows Failover Clustering still relies upon SCSI Persistent Reservations. Therefore, the storage must still be a Shared Storage type such as a SAN technology. Local disks or disks presented by a hypervisor may not be compatible. In Azure, the disks must be a Premium SSD size that supports sharing, even if only one VM is to be attached to it.

```